src/components/combat/Combat.js

```diff
@@ -0,0 +1,53 @@
+import combatStyle from './Combat.module.scss'
+import React from 'react'
+// import Button from '../button'
+import IzzyButton from 'components/izzyButton/'
+import { BUTTON_TYPE } from 'components/izzyButton/izzyButton'
+
+const Combat = () => {
+  const cardTexts = [
+    'Chronic Desease',
+    'Heart diseases',
+    'Pollution relataed sickness',
+    'Global warming potential health risk',
+  ]
+  return (
```

Comment:
What do think of leaving a blank line before returning the jsx?

```suggestion
  ]
  
  return (
```